lib/phlex/xml.rb

```diff
@@ -0,0 +1,315 @@
+# frozen_string_literal: true
+
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.0")
+	using Phlex::Overrides::Symbol::Name
+end
+
+module Phlex
+	class XML
```

Comment:
Can we rename this `SGML`? https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language

Reply:
I feel like `SGML` is weird, because no one knows this by heart. Also Nokogiri uses `XML` throughout their lib for representing XML and HTML nodes

Reply:
Users of Phlex probably don't need to know about the `SGML` class, right? I think `SGML` makes sense, and could see a future `Phlex::XML` to generate strict XML.

Reply:
Yes, I would like to have a `Phlex::XML` class for working with strict XML with support for namespaces.

Reply:
Fair enough 👍🏼